docs/requirements.txt

```diff
@@ -1,9 +1,7 @@
 sphinx
 sphinx_rtd_theme
-numpy>=1.7
 contextlib2
-cloudpickle>=0.3.1
```

Comment:
@jpchen can you confirm we no longer need cloudpickle or observations?

Reply:
yep LGTM. besides internal applications, i'm not sure which example was using cloudpickle in the first place...